machine/v1alpha1/types_openstack.go

```diff
@@ -100,6 +101,30 @@ type OpenstackProviderSpec struct {
 
 	// The subnet that a set of machines will get ingress/egress traffic from
 	PrimarySubnet string `json:"primarySubnet,omitempty"`
+
+	// FailureDomain defines the failure domain this machine will be
```

Comment:
Nit, first word should be the json tag
```suggestion
	// failureDomain defines the failure domain this machine will be
```